Feature profile

Aayam1o1 opened the pull request.

Aayam1o1 committed:
Completed owner profile

Used Signals to create owner profile automatically when car parking is created

Added button in index2.html to view owner profile through  modal

Aayam1o1 committed:
Added gitignore file

NiteshTuladhar commented on parking/core/models.py:
```diff
@@ -54,4 +54,18 @@ class ParkingDetail(models.Model):
+    
+    
+class OwnerProfile(models.Model):
+    owner_name_profile = models.OneToOneField(CarDetail, on_delete=models.CASCADE, blank = True, null = True, related_name="owner_name_profile")
```
incorrect naming, maybe naming it owned_car seems correct.

NiteshTuladhar replied:
related_name is not a necessity in a OneToOneFileld

NiteshTuladhar commented on parking/core/models.py:
```diff
@@ -54,4 +54,18 @@ class ParkingDetail(models.Model):
+    
+class OwnerProfile(models.Model):
+    owner_name_profile = models.OneToOneField(CarDetail, on_delete=models.CASCADE, blank = True, null = True, related_name="owner_name_profile")
+    owner_number_profile = models.CharField(max_length=30)
```
you can just write number_profile instead of owner_number_profile as it is the owner model.

NiteshTuladhar commented on parking/core/urls.py:
```diff
@@ -17,8 +17,9 @@
     # class based urls aba
-    path("index2", car_detail_view.as_view(), name='index2'),
-    path("more_car_detail", car_detail_more_view.as_view(), name='more_car_detial')
+    path("index2", CarDetailView.as_view(), name='index2'),
```
give appropriate naming for urls and also end the url with a '/'

NiteshTuladhar commented on parking/db.sqlite3:
add db.sqlite3 to the .gitignore as well

Aayam1o1 committed:
Added validation for the parking wing

Cannot create parking for if thw wing is already taken.

used sweetify for the toast to display message

Aayam1o1 committed:
deleted dbsqlite

Aayam1o1 committed:
fix: fixed gitignore for dbsqlite and fixed naming for urls and model name
